types/sparkly/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for sparkly 3.1
+// Project: https://github.com/sindresorhus/sparkly
+// Definitions by: BendingBender <https://github.com/BendingBender>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = sparkly;
+
+declare function sparkly(numbers: any[], options?: sparkly.Options): string;
```

Comment:
Ehhhh... I suppose so, but type-wise there's a big downside to an `any[]` - its so permissible that you can easily accidentally pass in entire lists of non-numbers, which would more likely than not indicate a bug in the code you've written (which the types would make it impossible to catch). `number | "" ` is a minimal sufficient member type to make the example compile, at least; is that better? I think it's unlikely someone makes an array of empty strings by accident.